core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -782,7 +782,7 @@ object KafkaConfig {
     "<code>message.max.bytes</code> (broker config) or <code>max.message.bytes</code> (topic config)."
   val ReplicaFetchWaitMaxMsDoc = "max wait time for each fetcher request issued by follower replicas. This value should always be less than the " +
   "replica.lag.time.max.ms at all times to prevent frequent shrinking of ISR for low throughput topics"
-  val ReplicaFetchMinBytesDoc = "Minimum bytes expected for each fetch response. If not enough bytes, wait up to replicaMaxWaitTimeMs"
+  val ReplicaFetchMinBytesDoc = "Minimum bytes expected for each fetch response. If not enough bytes, wait up to replica.fetch.wait.max.ms"
```

Comment:
@satishbellapu  Thanks for the PR.  Sorry for the delay. LGTM .  Similar to other docs, Can we place the config between <code>..</code> tags?

Reply:
@omkreddy done.